training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,716 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        # Donwload some mock data from a public bucket in us-east-1
</pre><p>Typo <code>Donwload</code> -&gt; <code>Download</code> ﻿ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>fixed</p>